apps/sms/views/shared/test/unit/activity_handler_test.js

```diff
@@ -864,9 +859,11 @@ suite('ActivityHandler', function() {
     });
 
     test('Should move to the composer and set activity', function(done) {
-      ActivityClient.on.withArgs('new-activity-request').yield(
-        numberActivityData
-      );
+      ActivityClient.on.withArgs('new-activity-request').yield({
+        number: '123',
+        body: 'foo'
+      });
+
       threadDeferred.reject(new Error('No thread for this test'));
```

Comment:
__nit:__ mm, maybe ```sinon.assert.notCalled(MessageManager.findThreadFromNumber)``` instead of ```threadDeferred.reject(new Error('No thread for this test'));```?